pkg/controller/cluster/main-controller.go

```diff
@@ -831,13 +951,6 @@ func (c *Controller) syncHandler(key string) error {
 		ss, err := c.statefulSetLister.StatefulSets(tenant.Namespace).Get(ssName)
 		if k8serrors.IsNotFound(err) {
 			klog.Infof("'%s/%s': Deploying pool %s", tenant.Namespace, tenant.Name, pool.Name)
-			// Check healthcheck for previous pool only if it's not a new setup,
-			// and check if they are online before adding this pool.
-			if addingNewPool && !tenant.MinIOHealthCheck(c.getTransport()) && len(tenant.Spec.Pools) > 1 {
```

Comment:
are you sure we don't want to wait for MinIO to be healthy?

Reply:
No we don't care @dvaldivia